src/config/konnectors.json

```diff
@@ -913,6 +913,31 @@
     },
     "source": "git://github.com/cozy/cozy-konnector-digiposte.git#build"
   },
+  {
+    "slug": "directenergie",
+    "name": "Direct Energie",
+    "editor": "Cozy Cloud",
```

Comment:
I am not sure about the editor here. Do we keep _Cozy Cloud_ even for contribrutors konnectors ?

Reply:
Right, this is a bad copy/paste